BootloaderCommonPkg/Include/Library/ElfLib.h

```diff
@@ -0,0 +1,70 @@
+/** @file
+  ELF library
+
+  Copyright (c) 2018, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __ELF_LIB_H__
+#define __ELF_LIB_H__
+
+#include <Uefi/UefiBaseType.h>
+#include <Library/MultibootLib.h>
```

Comment:
can we decouple Multiboot and ELF?   Why ELF needs to include MultiBoot?   Shouldn't be the other way ?

Reply:
I will do that.